backend/logic/courses/courses.js

```diff
@@ -29,6 +29,43 @@ var queryCourseByTag = async function queryCourseByTag(tag) {
   }
 };
 
+/**
+ * Adds a course into the database
+ * @author Mathieu Savoie
+ * @param {String} courseCode
+ * @param {String} title
+ * @param {String} departement
+ * @returns true if successful
+ * @throws error if MySQL connection failed
+ *         invalid format courses if JSON format is incorrect
+ *         false if insertion failed
+ */
+var addCourse = async (courseCode, title, departement, phasedOut) => {
```

Comment:
The default in the database for phasedOut is false so you dont explicitally have to specify it when inserting. Because of this I think it would be better if the front end was not forced to send a "phased out" param when adding a course into the db 
```suggestion
var addCourse = async (courseCode, title, departement) => {
```

Reply:
I tried following your instructions and looking online, but i get a bunch of errors... I have an assignment due in 1 hour that I have to finish, I can work on this during the break if necessary!

Reply:
no problem this is great!